MachObfuscator/SymbolsCollecting/ObfuscationSymbols+Building.swift

```diff
@@ -37,7 +38,17 @@ extension ObfuscationSymbols {
             .union(skippedSymbols.selectors)
         let blacklistSetters = blackListGetters.map { $0.asSetter }.uniq
 
-        let blacklistSelectors = (Array(blackListGetters) + Array(blacklistSetters)).uniq
+        let blacklistedSelectorsByRegex = userSelectors.filter { selector in
+            objcOptions.selectorsBlacklistRegex.contains(where: { regex in
+                selector.range(of: regex, options: .regularExpression, range: nil, locale: nil) != nil
```

Comment:
Reusing `NSRegularExpression` was promised to be very fast. I compared time measurements of using the `NSRegularExpression` and above method – and it seems like this method is 2x faster... Wow.

Reply:
I changed implementation to `NSRegularExpression`

Reply:
Wait. 😅 It was just a notice, not a change request. I measured performance and `NSRegularExpression` looks not as fast as `String.compare`. 😅 Your recent change has one advantage – MachObfuscator will fails on any malformed regex. This can stay as it is, but I am curious why you changed your mind? Did you misunderstand me?

Reply:
Heh:) I read too fast and blindly assumed that lower level (i.e. from ObjC) method will be faster.
As for performance gains: `NSRegularExpression` created ahead should allow the regex to be precompiled while for `String.range(of: regex,...` is it not so easy, but I don't know internals of `NSRegularExpression`.
And as you say, there is additional benefit of better syntax error checking.